staging/src/k8s.io/kubectl/pkg/cmd/diff/diff_test.go

```diff
@@ -636,3 +643,48 @@ func TestMasker(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateFields(t *testing.T) {
+	tests := []struct {
+		args         [][]string
+		enableAlphas []cmdutil.FeatureGate
+		expectedErr  string
+	}{
+		{
+			args: [][]string{
+				{"server-side", "false"},
+				{"force-conflicts", "true"},
+			},
+			enableAlphas: []cmdutil.FeatureGate{cmdutil.AutoServerSide},
+			expectedErr:  "error: --force-conflicts only works with --server-side",
+		},
+		{
```

Comment:
Suggestion: duplicate this case to show the different error message when the alpha is disabled.